tests/assembly/nvptx-kernel-abi/nvptx-kernel-args-abi-v7.rs

```diff
@@ -166,7 +166,7 @@ pub unsafe extern "ptx-kernel" fn f_f32_arg(_a: f32) {}
 pub unsafe extern "ptx-kernel" fn f_f64_arg(_a: f64) {}
 
 // CHECK: .visible .entry f_single_u8_arg(
-// CHECK: .param .align 1 .b8 f_single_u8_arg_param_0[1]
+// CHECK: .param .u8 f_single_u8_arg_param_0
```

Comment:
I presume this has an identical ABI.

Reply:
That might be correct. My worry here is that it goes against the "ptx-interoperability" docs. 

https://docs.nvidia.com/cuda/ptx-writers-guide-to-interoperability/index.html#aggregates-and-unions

Getting these kind of struct correct (per interop docs) was a big reason I couldn't get it "just right" when attempting to remove the PassMode::Direct

Reply:
The last commit reverts the ABI for this function back to what it was.